src/test/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/services/PlanServiceTest.kt

```diff
@@ -251,14 +251,14 @@ class PlanServiceTest {
         userDetails = userDetails,
       )
 
-      val planVersion = planService.signPlan(UUID.randomUUID(), signRequest)
+      val plan = planService.signPlan(UUID.randomUUID(), signRequest)
 
-      assertThat(planVersion.status).isEqualTo(CountersigningStatus.SELF_SIGNED)
+      assertThat(plan.currentVersion?.status).isEqualTo(CountersigningStatus.SELF_SIGNED)
```

Comment:
shouldn't the currentVersion.status be UNSIGNED?

Reply:
Of the new plan, yes, on the plan version that we just signed, I don't think so.